src/overmap_ui.cpp

```diff
@@ -689,6 +690,11 @@ static void draw_ascii(
             npc *npc_to_add = npc_to_get.get();
             followers.push_back( npc_to_add );
         }
+        if( !display_path.empty() ) {
```

Comment:
Because this is technically passed different data than the tile overmaps, future changes could accidentally knock them out of sync. 

It would be ideal to NOT have these be different, but the kludge in game.h is not something I want to rely on. That is why they are still different. But having them be different at all is another problem...